packages/nextjs/src/utils/config.ts

```diff
@@ -96,7 +98,7 @@ const injectSentry = async (origEntryProperty: EntryProperty, isServer: boolean)
   // `require()` on the resulting file when we're instrumenting the sesrver. (We can't use a dynamic import there
   // because that then forces the user into a particular TS config.)
   if (isServer) {
-    newEntryProperty[SERVER_SDK_INIT_PATH] = SENTRY_SERVER_CONFIG_FILE;
+    newEntryProperty[SERVER_SDK_INIT_PATH.slice(0, -3)] = SENTRY_SERVER_CONFIG_FILE;
```

Comment:
Can we add a comment about why we are stripping the file extension.

Reply:
Good point. Sure.